src/modules/TemplateImageLayer/LayerMaster.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, ManyToMany, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
```

Comment:
Remove `ManyToMany` and `OneToMany` from the imports as they are not being used in the file